editor/plugins/material_editor_plugin.cpp

```diff
@@ -197,6 +242,11 @@ MaterialEditor::MaterialEditor() {
 		box_switch->set_pressed(true);
 		sphere_switch->set_pressed(false);
 	}
+
+	first_enter = true;
+
+	rot_x = 0;
+	rot_y = 0;
```

Comment:
Those should be initialized in the header directly.

Reply:
Done for this and in mesh editors file too.